images-list

```diff
@@ -77,13 +81,16 @@ istio/proxyv2 rancher/istio-proxyv2 1.8.3
 istio/proxyv2 rancher/mirrored-istio-proxyv2 1.8.3
 jaegertracing/all-in-one rancher/jaegertracing-all-in-one 1.20.0
 jaegertracing/all-in-one rancher/mirrored-jaegertracing-all-in-one 1.20.0
+jenkins/jnlp-slave rancher/mirrored-jenkins-jnlp-slave 3.35-4
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.2.1
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.5.0
 jettech/kube-webhook-certgen rancher/mirrored-jettech-kube-webhook-certgen v1.2.1
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.3.0
 jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.4.0
 jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.3.0
 jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.4.0
+k8s.gcr.io/k8s-dns-node-cache rancher/mirrored-k8s-dns-node-cache 1.15.13 ?
```

Comment:
The source is k8s.gcr.io/dns/k8s-dns-node-cache based on this repo info:

https://github.com/kubernetes/dns